File: tagview/src/main/java/nesto/tagview/Tag.java
package nesto.tagview;

/**
 * Created on 2016/11/3.
 * By nesto
 */
public class Tag {
    public final String tag;
    public final Integer textColor;
    public final Integer backgroundColor;

    public Tag(String tag) {
        this.tag = tag;
        this.textColor = null;
        this.backgroundColor = null;
    }

    public Tag(String tag, Integer textColor, Integer backgroundColor) {
        this.tag = tag;
        this.textColor = textColor;
        this.backgroundColor = backgroundColor;
    }

}

File: tagview/src/main/java/nesto/tagview/Tag.kt
package nesto.tagview

import android.support.annotation.ColorInt

/**
 * Created on 2016/11/3.
 * By nesto
 */
class Tag {
    val tag: String
    val textColor: Int?
    val backgroundColor: Int?

    constructor(tag: String) {
        this.tag = tag
        this.textColor = null
        this.backgroundColor = null
    }

    constructor(tag: String, @ColorInt textColor: Int?, @ColorInt backgroundColor: Int?) {
        this.tag = tag
        this.textColor = textColor
        this.backgroundColor = backgroundColor
    }
}
